tests/Collection/LocaleCollectionTest.php

```diff
@@ -71,8 +72,8 @@ public function testGroupByCurrencyFormat()
 
         self::assertEquals(
             [
-                '€ 123.456.789,12' => new LocaleCollection([Locale::fromString('nl_NL')]),
-                '123.456.789,12 €' => new LocaleCollection([Locale::fromString('nl_BE'), Locale::fromString('de_DE')]),
+                '€ 123.456.789,12' => new LocaleCollection([Locale::fromString('nl_NL'), Locale::fromString('nl_BE')]),
```

Comment:
```
There was 1 failure:

1) MyOnlineStore\Common\Domain\Tests\Collection\LocaleCollectionTest::testGroupByCurrencyFormat
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
     '€ 123.456.789,12' => MyOnlineStore\Common\Domain\Collection\LocaleCollection Object (
         0 => MyOnlineStore\Common\Domain\Value\Locale Object (...)
-        1 => MyOnlineStore\Common\Domain\Value\Locale Object (...)
     )
     '123.456.789,12 €' => MyOnlineStore\Common\Domain\Collection\LocaleCollection Object (
         0 => MyOnlineStore\Common\Domain\Value\Locale Object (
             'regionCode' => MyOnlineStore\Common\Domain\Value\RegionCode Object (
-                'code' => 'DE'
+                'code' => 'BE'
             )
             'languageCode' => MyOnlineStore\Common\Domain\Value\LanguageCode Object (
-                'code' => 'de'
+                'code' => 'nl'
             )
         )
+        1 => MyOnlineStore\Common\Domain\Value\Locale Object (...)

/opt/packages/common-domain/tests/Collection/LocaleCollectionTest.php:78
```

Do you have another version of the intl library in your docker environment?

Reply:
Perhaps change it to some other currencly/locale combo's which are not subject to change. Or drop the function and test, since this implementation is not unit-testable is not used (the only usage still depends on the monolith version of this collection. It can be changed to a unit-testable service when that collection is replaced)

Reply:
It’s probably because I use a local version for phpunit. But I will look into dropping it then 🙂

Reply:
The formatting depends on the used INTL library version of PHP. Maybe it's worth to invest some time to look into [Symfony's implementation](https://packagist.org/packages/symfony/intl) (Application dependency) rather than the PHP one (OS/PHP dependency).

Reply:
I'll create a Jira improvement to look into and rewrite this function. A unit test like this should not be depend on an internal library version (imho). All it should check is whether locales are properly grouped

Reply:
Will revert the test for now then.